main/src/chat/chat_view_model.cpp

```diff
@@ -0,0 +1,84 @@
+#include "chat_view_model.h"
+
+template <typename T>
+struct IntervalRepeater {
```

Comment:
The more formal approach was abandoned, because the old way of passing a λ to env was not allocated anymore giving bad access calls, so I simplified and forgot to try the more foolproof way as compared to thread(λ).detatch();